wbooks/src/app/components/AppNavigator/config.tsx

```diff
@@ -2,20 +2,31 @@ import React from 'react';
 import { Image } from 'react-native';
 import backButton from '@assets/ic_back.png';
 import headerBackground from '@assets/bc_nav_bar.png';
+import SearchBar from '@components/SearchBar';
 import colors from '@constants/colors';
 
 import styles from './styles';
 
-const customHeader = {
+const sharedHeaderProperties = {
   headerStyle: {
     height: 90
   },
-  headerTintColor: colors.white,
-  headerBackImage: () => <Image style={styles.backButton} resizeMode="cover" source={backButton} />,
   headerBackground: () => <Image style={styles.headerImage} resizeMode="stretch" source={headerBackground} />,
   headerTitleStyle: {
     textTransform: 'uppercase'
   }
 };
 
-export default customHeader;
+const mainHeader = {
+  ...sharedHeaderProperties,
+  headerTintColor: colors.white,
+  headerBackImage: () => <Image style={styles.backButton} resizeMode="cover" source={backButton} />
+};
+
+export const searchHeader = {
+  ...sharedHeaderProperties,
+  headerLeft: () => null,
+  headerTitle: () => <SearchBar />
+};
+
+export default mainHeader;
```

Comment:
Why? Use `export const` when you have more than one object in a file

Reply:
Refactored the entire AppNavigator/config.tsx file